fhir-bulkimportexport-webapp/WebContent/WEB-INF/ibm-web-ext.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-ext
+	xmlns="http://websphere.ibm.com/xml/ns/javaee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://websphere.ibm.com/xml/ns/javaee http://websphere.ibm.com/xml/ns/javaee/ibm-web-ext_1_1.xsd"
+	version="1.1">
+
+	<reload-interval value="3"/>
+	<context-root uri="fhir-bulkimportexport-webapp" />
```

Comment:
is this the actual context root? in the server.xml?

Reply:
yes, same as in the sample server.xml